Automattic/wp-calypso
# Add Product Lightbox styling

#### Proposed Changes

This PR adds Styling for the LightBox for Product Cards. It has hardcoded content, so it would be the same for each card. The most accent is done for responsiveness and following the designs. 

#### Testing Instructions
Please test designs thorough on different screens (desktop/mobile, portrait/landscape). I would also appreciate any suggestions regarding improving CSS if needs one. Please don't look into content and it's behavior.

* Run `git fetch && git checkout update/jetpack-product-lightbox-product-data`
  * Run `yarn start-jetpack-cloud`
  * Go to http://jetpack.cloud.localhost:3000/pricing?flags=jetpack/pricing-page-product-lightbox

**OR**

* use the jetpack cloud live link and goto `/pricing?flags=jetpack/pricing-page-product-lightbox

* On product cards click on the question mark button at the bottom right of the card and observe the styling of the lightbox.



https://user-images.githubusercontent.com/60262784/187956303-6e562582-1efa-4cb5-9814-18e080db4b46.mov


#### Pre-merge Checklist






Related to #
1202796695664022-as-1202900224232205/f

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202900224232205
